src/netif.c

```diff
@@ -2472,16 +2472,31 @@ static void lcore_job_xmit(void *args)
 }
 
 static int timer_sched_interval_us;
+static rte_atomic64_t ticks;
+static int64_t ticks_tbl[DPVS_MAX_LCORE] = { 0 };
+
+static void lcore_job_slave_timer_manage(void *args) 
+{
+    int64_t local_ticks;
+    lcoreid_t cid = rte_lcore_id();    
+
+    local_ticks = rte_atomic64_read(&ticks);
+    while (local_ticks != ticks_tbl[cid]) {
+        rte_timer_manage();
+        ticks_tbl[cid]++;
```

Comment:
`rte_timer_manage` may be called more than once for each loop. If so, it's not fair for other jobs.